CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+## Welcome!
+
+We're so glad you're thinking about contributing to an 18F open source project! If you're unsure about anything, just ask -- or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.
```

Comment:
I vote for WW. Then the text is also formatted for people viewing these files in a terminal. It does not affect the formatting in GH which ignores WW, for example see https://github.com/18F/identity-sp-rails/blob/license-contributing/LICENSE.md vs https://raw.githubusercontent.com/18F/identity-sp-rails/license-contributing/LICENSE.md